docs/low-code-spark/gems/machine-learning/ml-pinecone-lookup.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 3
+title: Pinecone Lookup
+id: ml-pinecone-lookup
+description: Lookup a vector embedding from a Pinecone Database
+tags: [generative-ai, machine-learning, llm, pinecone, openai]
+---
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/019qt3csq9?seo=false?videoFoam=true" title="Getting Started With SQL Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+The Pinecone Lookup Gem identifies content that is similar to a provided vector embedding. The Gem operation, input, and output are breifly described below.
+
+**Gem operation:** The Pinecone Lookup Gem calls the Pinecone API and returns a set of IDs with highest similarity to the provided embedding.
+
+**Input:** This Gem requires an embedding as input. The embedding is provided by a foundational model like [OpenAI](https://platform.openai.com/docs/introduction).
+
+**Output:** This Gem outputs an array of IDs with corresponding similarity scores.
+
+![Input and Output](./img/pinecone_lookup_input_output.png)
+
+Now let’s understand the Gem Configuration, Input, and Output in detail.
+
+### Gem Configuration
```

Comment:
Everywhere else, we're calling this "Parameters" - let's stick to the same naming convention.